src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/util/AnsiCodes.java

```diff
@@ -0,0 +1,29 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.util;
+
+public class AnsiCodes {
+
+  private AnsiCodes() {}
```

Comment:
Ist zwar nur kosmetisch, aber bin mir ziemlich sicher das die win cmd kein ansi unterstützt.

Reply:
in der PowerShell hats geklappt ;)

Reply:
Ja, die unterstützt ansi. Wie gesagt nur kosmetisch, kein Grund dein pr nicht zu approven.